handbook/engineering/security/index.md

```diff
@@ -10,6 +10,10 @@ Make Sourcegraph secure by design.
 - #security channel Slack.
 - [@sourcegraph/security](https://github.com/orgs/sourcegraph/teams/security) on GitHub.
 
+## Philosophy
+
+At Sourcegraph we think that security is an enabler for the business. Sourcegraph is committed to proactive security, addressing vulnerabilities in a timely manner. We approach security with a can-do philosophy, and look to achieve product goals while maintaining a positive posture, and increasing our security stance over time.
```

Comment:
```suggestion
At Sourcegraph we think that security is an enabler for the business. Sourcegraph is committed to proactive security, and addressing vulnerabilities in a timely manner. We approach security with a can-do philosophy, and look to achieve product goals while maintaining a positive posture, and increasing our security stance over time.
```

Reply:
This is great and I think it should be inlined immediately after the page title (not in a separate `## Philosophy` section) and we can remove `## Vision` too.